admin/modules/tools/tasks.php

```diff
@@ -87,21 +87,21 @@ function check_time_values($value, $min, $max, $return_type)
 			$errors[] = $lang->error_invalid_task_file;
 		}
 
-		$mybb->input['minute'] = check_time_values($mybb->input['minute'], 0, 59, 'string');
+		$mybb->input['minute'] = isset($mybb->input['minute']) ? check_time_values($mybb->input['minute'], 0, 59, 'string') : false;
```

Comment:
Instead of a ternary operator, `$mybb->input` usage could be replaced with `$mybb->get_input()` that does the necessary checks and returns a default value:
https://github.com/mybb/mybb/blob/0bf8fdb90008c82e20cd6350ce74928e08a6bdda/inc/class_core.php#L464